core/src/main/scala/org/http4s/rho/Result.scala

```diff
@@ -72,38 +73,88 @@ F[_],
 object Result {
 
   /** Result type with completely ambiguous return types */
-  type BaseResult[F[_]] = Result[F, Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any,Any]
+  type BaseResult[F[_]] = Result[F, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any, Any]
 
   /** Result with no inferred return types */
-  type TopResult[F[_]]  = Result[F, Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing ,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing,Nothing]
+  type TopResult[F[_]] = Result[F, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing, Nothing]
 
   /** Existential result type */
-  type ExResult[F[_]]   = Result[F, _,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_,_]
+  type ExResult[F[_]] = Result[F, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _]
 }
 
-import org.http4s.rho.Result._
-
 trait ResultSyntaxInstances[F[_]] {
-  implicit class ResultSyntax[T >: Result.TopResult[F] <: BaseResult[F]](r: T) extends ResponseOps[F] {
-    override type Self = T
+  implicit class ResultSyntax[R >: Result.TopResult[F] <: Result.BaseResult[F]](r: R) {
+    type Self = R
+
+    private val resp: Response[F] = r.resp
+
+    //def mapK[G[_]](f: ~>[F, G]): R[G] = Result(resp.mapK(f)).asInstanceOf[R[G]]
```

Comment:
Any suggestions on how to implement this?  I played around with changing the `R` here but wasn't able to get something that would resolve implicitly...

Reply:
Do you have a need for that, or are you trying to keep parity for eventual reunification in a typeclass?  In other words, how hard do we need to try?

Reply:
I don't have an immediate need.  Just trying to keep some party.  It doesn't have to make it in this PR.